v0/destinations/candu/transform.js

```diff
@@ -14,9 +15,23 @@ const responseBuilder = (body, { Config }) => {
   payload.context = {
     source: "RudderStack"
   };
+  if (payload.userId) {
+    payload.userId = `${payload.userId}`;
```

Comment:
Do we require to handle the case for numeric userId if userId is provided inside traits too?

Reply:
No